components/tools/OmeroWeb/omeroweb/webgateway/templatetags/common_filters.py

```diff
@@ -50,8 +50,9 @@ def hash(value, key):
 
 
 @register.filter
-def random_if_none(value):
```

Comment:
Removing this could conceivably break someone, no? (even if **really** unlikely) Can you restore `random_if_none` beside `random_if_minus_one`